numpyro/examples/baseball.py

```diff
@@ -195,8 +194,8 @@ def main(args):
                                partially_pooled,
                                partially_pooled_with_logit,
                                )):
-        rng, rng_predict = random.split(random.PRNGKey(i))
-        zs = run_inference(model, at_bats, hits, random.PRNGKey(i), args)
+        rng, rng_predict = random.split(random.PRNGKey(i + 1))
```

Comment:
I want to make sure that this PRNGKey is different from the one is used in running hmc_init